Examples/Swift/LocationPrivacyExample.swift

```diff
@@ -0,0 +1,46 @@
+import Mapbox
+
+@objc(LocationPrivacyExample_Swift)
+
+class LocationPrivacyExample_Swift: UIViewController, MGLMapViewDelegate {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        mapView.userTrackingMode = .follow
+
+        view.addSubview(mapView)
+    }
+
+    /**
+        In order to enable the alert that requests temporary precise location, please add the following key to your info.plist
+        Privacy - Location Temporary Usage Description Dictionary
```

Comment:
Nit: I'd surround `Privacy - Location Temporary Usage Description Dictionary` by backticks to make it clear what the key is.